<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountClientsWidget</class>
 <widget class="QWidget" name="AccountClientsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>435</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>752</width>
    <height>435</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>190</width>
     <height>207</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QGroupBox" name="searchGroupBox">
      <property name="title">
       <string>Rechercher Un Compte Par : </string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="firstRadioButton">
           <property name="text">
            <string>Numero De Compte</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="secondRadioButton">
           <property name="text">
            <string>Nom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="thirdRadioButton">
           <property name="text">
            <string>Prenom</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="inputLabel">
             <property name="text">
              <string>Numero de Compte:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="inputLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="searchButton">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Recherche Le Compte a Modifer</string>
             </property>
             <property name="text">
              <string>Rechercher</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>21</y>
     <width>501</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Selectionner Le Compte A Modifier</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="cancelButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>205</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Modifier Les Infromations :</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>nom:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Prenom:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>age:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>tel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_4"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>adresse:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_5"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="comboBox_2">
          <item>
           <property name="text">
            <string>Etudiant (e)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sans-Travail</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fonctionnaire</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>183</width>
          <height>0</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Sauvegarder les Informations Modifie</string>
        </property>
        <property name="text">
         <string>Sauvegarder</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
